lib/vehicle_factory.rb

```diff
@@ -0,0 +1,34 @@
+class VehicleFactory
+    attr_accessor :vehicle_lot
```

Comment:
Do we need this to be an accessor? We want to not expose methods unless they're needed.